src/service/standard-actions-impl.js

```diff
@@ -68,6 +68,14 @@ export class StandardActions {
     /** @const @private {!./viewport/viewport-impl.Viewport} */
     this.viewport_ = Services.viewportForDoc(ampdoc);
 
+    const meta =
```

Comment:
Maybe a bit of documentation here would be good?

// Cache the whitelist of allowed AMP actions (if provided). The whitelist is stored in a meta tag named `amp-action-whitelist`.

Or something like that

Reply:
Done